cypress/fixtures/suite/backoffice/file-manager-attachment/static-file-list.json

```diff
@@ -0,0 +1,12 @@
+{
+  "defaultPassword": "change123",
+  "rootUser": {
+    "firstName": "Admin",
+    "lastName": "Spryker",
+    "username": "admin@spryker.com"
```

Comment:
do we need it? we should use dynamic fixtures for users

Reply:
removed